config.py

```diff
@@ -1,7 +1,13 @@
 import logging
 import os
 
-COBOT_ROOT = os.environ.get('COBOT_ROOT', os.getcwd())
+# Deprecate COBOT_ROOT
+if 'COBOT_ROOT' in os.environ and 'BOT_ROOT' not in os.environ:
+    logging.warning(
+        'Deprecation warning: environment variable {} is replaced by {}.'
```

Comment:
--> "Environment variable {} is deprecated, use {} instead."

For "Deprecation *warning*", the word "warning" would occur twice in the same log entry, because you also log with `logging.warning` ;)